upstream_utils/concurrentqueue.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import os
+import shutil
+
+from upstream_utils import Lib
+
+
+def copy_upstream_src(wpilib_root):
+    wpiutil = os.path.join(wpilib_root, "wpiutil")
+
+    headers = [
+        "blockingconcurrentqueue.h",
```

Comment:
According to the issue we don't need the blocking version
```suggestion
```

Reply:
That was added after i opened the PR- thanks

Reply:
I'd be fine with the blocking version. Might be useful at some point.

Reply:
That's why I included it originally, it didn't seem harmful and I didn't see peter's note. Should I add it back in?

Reply:
Ah. Yeah it should be fine to not have it then.